packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -279,8 +275,8 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
     }
 
     /// @notice Getter for the address of the shared lockbox.
-    function sharedLockbox() public view returns (address) {
-        return address(SHARED_LOCKBOX);
+    function sharedLockbox() public view returns (ISharedLockbox) {
```

Comment:
Since this getter adds a call now, I think it makes sense to have a new small test covering it